readme.md

```diff
@@ -64,6 +95,10 @@ Hello, Node!
 
 * **Your Answer:**
 
+`writeFile` might create a new file.  It overwrites the file if it exists.  Usually use this.
+
+`appendFile` will not create a new file.
```

Comment:
It will actually create the new file if it doesn't exist. The difference here is that `.appendFile()` will just add to the end of a file.